src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/RazorHoverEndpoint.cs

```diff
@@ -73,9 +73,9 @@ protected override IDelegatedParams CreateDelegatedParams(VSHoverParamsBridge re
         }
 
         /// <inheritdoc/>
-        protected override async Task<VSInternalHover> HandleDelegatedResponseAsync(VSInternalHover response, DocumentContext documentContext, CancellationToken cancellationToken)
+        protected override async Task<VSInternalHover?> HandleDelegatedResponseAsync(VSInternalHover? response, DocumentContext documentContext, CancellationToken cancellationToken)
         {
-            if (response.Range is null)
+            if (response is null || response.Range is null)
```

Comment:
I think (hope) there are some nullable attributes that we can use to avoid this, but that can be a follow up at some point